.circleci/config.yml

```
@@ -46,7 +46,7 @@ jobs:
       go_version:
         type: string
     environment:
-      CI_E2E_FILENAME: "fafa8862/rel-nightly"
+      CI_E2E_FILENAME: "feature-avm-box"
```

```suggestion
      CI_E2E_FILENAME: "feature-avm-box"
```
Before merging: revert this to a more up to date E2E tarball that is from `go-algorand/master` but includes boxes